src/backend/storeManagers/legendary/library.ts

```diff
@@ -744,6 +745,57 @@ export async function getGameSdl(
 }
 
 /**
+ * Toggles the EGL synchronization on/off based on arguments
+ * @param path_or_action On Windows: "unlink" (turn off), "windows" (turn on). On linux/mac: "unlink" (turn off), any other string (prefix path)
+ * @returns string with stdout + stderr, or error message
+ */
+export async function toggleGamesSync(path_or_action: string) {
+  const isWindows = platform() === 'win32'
```

Comment:
We already have a `isWindows` export in `backend/constants.ts`, might want to use that

Reply:
changed